cdp_scrapers/legistar_utils.py

```diff
@@ -243,6 +245,100 @@ def get_legistar_person(
     return person
 
 
+def get_legistar_matter(
+    client: str,
+    matter_id: Union[int, str],
+    use_person_cache: bool = False,
+) -> Optional[Dict[str, Any]]:
+    """
+    Return information for a single legistar Matter in JSON.
+
+    Parameters
+    ----------
+    client: str
+        Which legistar client to target. Ex: "seattle"
+    matter_id: Union[int, str]
+        Unique ID for this matter in the legistar municipality
+    use_person_cache: bool
+        Pass to get_legistar_person(use_cache=use_person_cache)
+
+    Returns
+    -------
+    matter: Dict[str, Any]
+        legistar API Matter
+    """
+    resp = requests.get(
+        (LEGISTAR_MATTER_BASE + "/{matter_id}").format(
+            client=client, matter_id=matter_id
+        )
+    )
+    if resp.status_code != 200:
+        log.debug(
+            f"{resp.status_code} ({resp.reason}) from querying for "
+            f"Legistar Matter with MatterId = {matter_id}"
+        )
+        return None
+    matter: Dict[str, Any] = resp.json()
+
+    # Person JSON for this matter's sponsors
+    matter[LEGISTAR_MATTER_SPONSORS] = reduced_list(
```

Comment:
I thought this was some built-in python method at first and was kinda confused lol. I think this list comprehension is a little awkward to read but I'm fine with it

Reply:
That's a good point. I will change this. Readability is important for us.